File: src/api/src/Scool.Domain/Courses/Model/Course.cs
using System;
using Volo.Abp.Domain.Entities;

namespace Scool.Domain.Courses.Model
{
    public class Course : Entity<Guid>
    {
        public string Name { get; set; }
        public string Description { get; set; }
        public DateTime StartTime { get; set; }
        public DateTime FinishTime { get; set; }
    }
}

File: src/app/common/interfaces/Course.ts

export namespace Course {
  export interface CourseDto {
    id: string;
    name: string;
    description?: string;
    startTime: Date;
    finishTime: Date;
  }
  
  export interface CreateOrUpdateCourseDto {
    name: string;
    description: string;
    startTime: Date;
    finishTime: Date;
  }
}

